source/PluginDev/Assets/GooglePlayGames/Platforms/Native/NativeClient.cs

```diff
@@ -288,10 +288,26 @@ private static void LaunchBridgeIntent(IntPtr bridgedIntent)
         }
         
         private AndroidJavaObject GetApiClient(GameServices services) {
-            return JavaUtils.JavaObjectFromPointer(C.InternalHooks_GetApiClient(services.AsHandle()));
+            //return JavaUtils.JavaObjectFromPointer(GooglePlayGames.Native.Cwrapper.InternalHooks.InternalHooks_GetApiClient(services.AsHandle()));
+            using (var currentActivity = GetActivity()) {
+                using (AndroidJavaClass jc_plus = new AndroidJavaClass("com.google.android.gms.plus.Plus")) {
+                    using (AndroidJavaObject jc_builder = new AndroidJavaObject("com.google.android.gms.common.api.GoogleApiClient$Builder",currentActivity)) {
+                        jc_builder.Call<AndroidJavaObject> ("addApi", jc_plus.GetStatic<AndroidJavaObject>("API"));
+                        jc_builder.Call<AndroidJavaObject> ("addScope", jc_plus.GetStatic<AndroidJavaObject>("SCOPE_PLUS_LOGIN"));
+                        AndroidJavaObject client = jc_builder.Call<AndroidJavaObject> ("build");
+                        client.Call ("connect");
+                        while(!client.Call<bool>("isConnected"))
```

Comment:
That's what I was talking about https://github.com/playgameservices/play-games-plugin-for-unity/pull/641#issuecomment-126481543

Without adding the API and Scope, I get

    07-30 21:45:52.266  24174-24193/? I/Unity﹕ AndroidJavaException: java.lang.IllegalStateException: GoogleApiClient is not configured to use the Plus.API Api. Pass this into GoogleApiClient.Builder#addApi() to use this feature.
            at UnityEngine.AndroidJNISafe.CheckException () [0x00000] in <filename unknown>:0
            at UnityEngine.AndroidJNISafe.CallStringMethod (IntPtr obj, IntPtr methodID, UnityEngine.jvalue[] args) [0x00000] in <filename unknown>:0
            at UnityEngine.AndroidJavaObject._Call[String] (System.String methodName, System.Object[] args) [0x00000] in <filename unknown>:0
            at UnityEngine.AndroidJavaObject.Call[String] (System.String methodName, System.Object[] args) [0x00000] in <filename unknown>:0
            at GooglePlayGames.Native.NativeClient.GetEmail () [0x00000] in <filename unknown>:0
            at GooglePlayGames.Native.NativeClient.GetAccessToken () [0x00000] in <filename unknown>:0
            at GooglePlayGames.PlayGamesPlatform.GetAccessToken () [0x00000] in <filename unknown>:0
            at MainScript.setToken () [0x00000] in <filename unknown>:0

When looking for the id token, or the access token.